src/content/chainlink-automation/overview/getting-started.mdx

```diff
@@ -35,27 +35,27 @@ Click the tabs below to use Chainlink Automation with each type of trigger:
 
       1. Navigate to the [Chainlink Automation app](https://automation.chain.link/) and connect to **Arbitrum Sepolia** in the top dropdown menu.
 
-      2. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Sepolia testnet LINK [here](https://automation.chain.link/arbitrum-sepolia).
+      1. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Sepolia testnet LINK [here](https://automation.chain.link/arbitrum-sepolia).
 
-      3. Click **Register new Upkeep** and select **Time-based** trigger.
+      1. Click **Register new Upkeep** and select **Time-based** trigger.
 
-      4. Under _Target contract address_, enter `0x698a3a4B0a98dda279795521Bd6f63cdBE605125`. This is a simple counter contract that will just increment with each call. Source code is [here](https://sepolia.arbiscan.io/address/0x083935210524c0A8922ec610d1063Aa0A54d9d70#code).
+      1. Under _Target contract address_, enter <Address contractUrl="https://sepolia.arbiscan.io/address/0x083935210524c0A8922ec610d1063Aa0A54d9d70" />. This is a simple counter contract that increments with each call. View the source code [here](https://sepolia.arbiscan.io/address/0x083935210524c0A8922ec610d1063Aa0A54d9d70#code).
 
-      5. In the _Contract call_ section, enter `addInteger` under _Target function_. In the _Function inputs_ section, enter a number to increment by under _intToAdd_. Then click **Next**.
+      1. In the _Contract call_ section, enter `addInteger` under _Target function_. In the _Function inputs_ section, enter a number to increment by under _intToAdd_. Then click **Next**.
 
          <ClickToZoom src="/images/automation/getting-started/gs_tbu_contract_address.png" />
 
-      6. Now specify the time schedule, for example every 5 minutes. Paste the cron expression `*/5 * * * *` under _Cron expression_ or select one of the example timers. Then click **Next**.
+      1. Specify the time schedule, for example every 5 minutes. Paste the cron expression `*/5 * * * *` under _Cron expression_ or select one of the example timers. Then click **Next**.
```

Comment:
Import `CopyText` then add copy/paste component like this:
`<CopyText text="*/5 * * * *" code/>`